bot/exts/help_channels/_channel.py

```diff
@@ -48,7 +47,18 @@ async def _close_help_post(closed_post: discord.Thread, closing_reason: _stats.C
     closed_post = await get_or_fetch_channel(bot.instance, closed_post.id)
 
     embed = discord.Embed(description=CLOSED_POST_MSG)
-    embed.set_author(name=f"{POST_TITLE} closed", icon_url=CLOSED_POST_ICON_URL)
+    close_title = "Python help channel closed"
+    if closing_reason == _stats.ClosingReason.CLEANUP:
+        close_title += " as OP left server"
+    elif closing_reason == _stats.ClosingReason.COMMAND:
+        close_title += f" by OP with {constants.Bot.prefix}close"
+    elif closing_reason == _stats.ClosingReason.INACTIVE:
+        close_title += " for inactivity"
+    elif closing_reason == _stats.ClosingReason.NATIVE:
+        close_title += " by OP"
```

Comment:
Moderators can close help channels too. 
Do you think it's worth distinguishing between an OP closing the post and a moderator closing the post?

Reply:
Se most recent commits